plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -1,12 +1,14 @@
 # Creating and Managing Custom Tables
 
-Welcome to Module 7 of the Intermediate Plugin Developer learning pathway. In this lesson, we will explore how to create, manage, and delete custom database tables in WordPress. As an intermediate developer, this skill is crucial for extending WordPress functionality beyond standard post types and taxonomies.
+As a plugin developer, knowing how to extend WordPress functionality beyond standard post types and taxonomies is a valuable skill to master.
+
+In this lesson, we will explore how to create, manage, and delete custom database tables in WordPress. 
 
 By the end of this lesson, you’ll understand why you might need a custom table, how to create and version the table schema, and how to safely remove a table when it’s no longer needed.
 
 ## Why Create a Custom Table?
 
-In most cases, WordPress's default tables are already enough to store your plugin's data. You can use the posts table to manage different types of content, store user meta to further personalize users' experiences, manage custom settings in the options table, and so much more!
+In most cases, WordPress's default tables are already enough to store your plugin's data. You can use [custom post types](link to cpt lesson) to manage different types of content, store user meta to further personalize users' experiences, manage custom settings in the options table, and so much more!
```

Comment:
Can we link to the other relevant lessons, too? I did see we have the other lessons and wanted to reference them without being wordy. I think the hyperlinks are a great way to do this! Or even just a relevant documentation could be helpful, too.

Reply:
Yes, I agree, linking to relevant lessons is always a good idea.

Reply:
Resolving this comment since it's now marked Outdated, but I have added another more specific comment to follow up on this.